logger/logger.js

```diff
@@ -0,0 +1,11 @@
+const { MCLogger } = require("@map-colonies/mc-logger");
+const service = require('../package.json');
+
+const config = {
+  level: "info",
```

Comment:
get config settings from config file (json)

Reply:
fixed in  001241c

Reply:
fixed in  001241c